src/styles/levelSelect.scss

```diff
@@ -25,21 +33,39 @@ $level-select-background: rgba($color: #fff, $alpha: 0.7);
   border-radius: 2em;
   display: inline-flex;
   flex-direction: row;
+  grid-column: 1;
+  grid-row: 2;
   justify-content: center;
-  margin: 0 -1.8em;
   padding: 0.3em 4em;
+  transition: all 0.1s ease;
+}
+
+@media screen and (max-width: 1200px) {
+  .dropdown-control {
+    padding: 0.3em 2em;
+  }
 }
   
 .dropdown-control:hover {
   background-color: $level-select-hover;
   cursor: pointer;
 }
+
+.dropdown-placeholder {
+  white-space: nowrap;
+}
   
+.dropdown-menu {
+  grid-column: 1;
+  grid-row: 1;
+}
+
 //dropdown-option classes
 .dropdown-option {
   background-color: $level-select-background;
-  margin: 0 -1.8em;
-  padding: 0.1em 0;
+  border-top: solid 0.1px white;
+  padding: 0.1em 0
+  // transition: all 0.05s ease;
```

Comment:
i commented it out for now because i haven't figured out exactly what the styling should look like for the level select. @nendow02 said to wait to do css tweaks until later